docs/en/week03/03-3.md

```diff
@@ -112,12 +108,13 @@ In this notebook, we train a multilayer perceptron (FC network) and a convolutio
 <center> <img src="{{site.baseurl}}/images/week03/03-3/Figure 8 Instances from the Original MNIST Dataset.png" width="350px" /><br>
 <b>Figure 8:</b> Instances from the Original MNIST Dataset </center>
 
-Before training, we normalize our data so that the initialization of the network will match our data distribution (very important!). Also, make sure that the following five operations/steps are present in your training: -
-1. Feeding data to the model
-2. Computing the loss
-3. Cleaning the cache of accumulated gradients with `zero_grad()`
-4. Computing the gradients
-5. Performing a step in the optimizer method
+Before training, we normalize our data so that the initialization of the network will match our data distribution (very important!). Also, make sure that the following five operations/steps are present in your training:
+
+ 1. Feeding data to the model
+ 2. Computing the loss
+ 3. Cleaning the cache of accumulated gradients with `zero_grad()`
+ 4. Computing the gradients
+ 5. Performing a step in the optimizer method
 
```

Comment:
```suggestion
Before training, we normalize our data so that the initialization of the network will match our data distribution (very important!). Also, make sure that the following five operations/steps are present in your training: -
1. Feeding data to the model
2. Computing the loss
3. Cleaning the cache of accumulated gradients with `zero_grad()`
4. Computing the gradients
5. Performing a step in the optimizer method
```